main.py

```
@@ -0,0 +1,15 @@
+"""
+本代码由[Tkinter布局助手]生成
+官网:https://www.pytk.net
+QQ交流群:905019785
+在线反馈:https://support.qq.com/product/618914
+"""
+# 导入布局文件
+from ui import Win as MainWin
```

Pylint (3.3.7)

[error] No name 'Win' in module 'ui'

(E0611)